flytekit/remote/remote.py

```diff
@@ -939,6 +941,9 @@ def _version_from_hash(
         for s in additional_context:
             h.update(bytes(s, "utf-8"))
 
+        if default_inputs:
+            h.update(cloudpickle.dumps(default_inputs))
```

Comment:
If the default inputs are big in-memory objects, then `clockpickle.dumps` will add more overhead. I do not think this is the case for Flyte.

Do you see use cases where the default inputs are big in-memory objects?

Reply:
That's not the common use case IMO.